.github/workflows/ci.yml

```diff
@@ -14,6 +14,37 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  precheckin:
+    runs-on: ubuntu-latest
```

Comment:
I just noticed that this can cause issues once the ubuntu-latest changes because we are specifically downloading llvm for `jammy` which is ubuntu-22.04. I would suggest that we should fix this to `ubuntu-22.04`, I will make sure to fix this in clad too.